### DISCUSSION TITLE
File has a  virus  tested in Virus Total

### DISCUSSION THREAD

**Discussion by gsbnlda** [CookieJarApps/SmartCookieWeb-Preview#98] [General]
This file has a virus , app-arm64-v8a-release.app , https://github.com/CookieJarApps/SmartCookieWeb-Preview/releases/tag/16.0  

 Tested : - https://www.virustotal.com/gui/file/146e5c9f188a41579c012d58e3a5cd05e35576a3213ce514b5108f5f72f7fd44

**Comment by CookieJarApps** [comment]
Hi, unfortunately some of the virus engines (in this case one) on VirusTotal detect false positives on some APKs - a few engines used to flag the WebView-based SmartCookieWeb. The only thing I can do is reach out to the company behind the antivirus flagging it and alert them to the false positive (which I did for SmartCookieWeb, which is now correctly flagged as safe by all engines). I'm not sure how to reach out to the specific company in this case, but I'll look into it. This is definitely a false positive though, which is backed up by how just one engine is flagging the file and the rest mark it as safe.
